src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -129,6 +138,36 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             const project = this.props.recentProjects.find((project) => project.id === projectId);
             await this.props.actions.loadProject(project);
         }
+
+        let modelPath = "";
+        if (this.props.project.activeLearningSettings.modelPathType === ModelPathType.Coco) {
+            if (isElectron()) {
+                const appPath = this.getAppPath();
+
+                if (Env.get() !== "production") {
+                    modelPath = appPath + "/cocoSSDModel";
+                } else {
+                    modelPath = appPath + "/../../cocoSSDModel";
+                }
+            } else {
+                modelPath = "https://jmangiadiag.blob.core.windows.net/vottcontainer";
+            }
+        } else if (this.props.project.activeLearningSettings.modelPathType === ModelPathType.File) {
+            if (isElectron()) {
+                modelPath = this.props.project.activeLearningSettings.modelPath;
+            }
+        } else {
+            modelPath = this.props.project.activeLearningSettings.modelUrl;
+        }
+
+        // Load TensorFlow.js Model
+        this.model = new ObjectDetection();
+        const infoId = toast.info("Loading model...", { autoClose: false });
```

Comment:
we should pull this string from the localization strings.

Reply:
Completely agree but there are many other cases like this.  For example the "Start/Finish exporting" toast and other.